bundle/config/bundle.go

```diff
@@ -5,7 +5,7 @@ type Terraform struct {
 }
 
 type Bundle struct {
-	Name string `json:"name"`
+	Name string `json:"name,omitempty"`
```

Comment:
I don't think having an empty bundle name is allowed, why this change then?

Reply:
Removed the tag. Good point. Originally added it for consistancy just because it's present everywhere else too.